api/src/services/opportunities_v1/search_opportunities.py

```diff
@@ -99,7 +99,7 @@ def _get_sort_by(pagination: PaginationParams) -> list[tuple[str, SortDirection]
 
     # Add a secondary sort for relevancy to sort by post date (matching the sort direction)
     if pagination.order_by == "relevancy":
-        sort_by.append((_adjust_field_name("post_date"), pagination.sort_direction))
+        sort_by.append((_adjust_field_name("post_date"), SortDirection.DESCENDING))
```

Comment:
Why did we change this? This will make it so reversing the sorting doesn't reverse the results.

Reply:
As far as I can tell there will be no way to hit this branch of the code with the `sort_direction` set to ascending. We are not going to allow the user to sort by ascending relevancy in the UI, as showing the least relevant results first doesn't seem like a common or expected use case. Since the sort_direction would come from the primary sort option, in this case relevancy, that would mean that the secondary sort would always come out as descending in this case.

That being the case we could still leave this code as is and it would always come out as descending, no impact to the user or the functionality of the code. However, I think it is could be best to hardcode the secondary sort behavior for relevancy to descending post date here for clarity and to avoid confusion. When I looked at the code I had to do a bunch of thinking to figure out if we could end up in a case where the secondary sort would actually be post date ascending, so if our intention is for that to not be possible we may as well hard code it, is where I ended up.

This is still a draft though, so I may change my mind before I open for review.

Reply:
I agree that I'm not sure in what scenario a user would want to sort by score/relevancy ascending, but at least on the API side I'd prefer we leave it as it was - the API can be used directly as well without the front-end so we need to consider the behavior there.

Likely what will make sense long-term is to remove this and just allow for the API request to specify a secondary sort directly.

Reply:
If we wanted to restrict relevancy sort to not allow ascending at all on the API, that would also be reasonable I think - although that change would require a little more work in the API schema to setup.

Reply:
I went ahead and reverted the change. I think it'd be better to be explicit about this sort behavior to avoid confusion, but that can wait for another ticket